index.html

```diff
@@ -417,12 +417,12 @@
       </div>
       <div>
         <pre class="idl">dictionary RTCIceCandidate {
-             DOMString              foundation;
-             unsigned long          priority;
-             DOMString              ip;
-             RTCIceProtocol         protocol;
-             unsigned short         port;
-             RTCIceCandidateType    type;
+             required DOMString              foundation;
+             required unsigned long          priority;
+             required DOMString              ip;
+             required RTCIceProtocol         protocol;
+             required unsigned short         port;
+             required RTCIceCandidateType    type;
              RTCIceTcpCandidateType tcpType;
              DOMString              relatedAddress = "";
```

Comment:
If this field is optional anyway, making it empty string by default shouldn't be necessary I guess. If removed, this should also be reflected in the description of `relatedAddress`.